docker-compose/basic/otel-collector-config.yaml

```diff
@@ -28,10 +31,10 @@ service:
   extensions: [health_check]
   pipelines:
     traces:
-      receivers: [otlp,otlp/2]
+      receivers: [otlp, otlp/2]
       processors: [batch]
       exporters: [jaeger]
     metrics:
-      receivers: [ otlp,otlp/2 ]
-      processors: [ ]
-      exporters: [ logging ]
\ No newline at end of file
+      receivers: [otlp, otlp/2]
+      processors: []
+      exporters: [logging, prometheus]
```

Comment:
can you please create a new docker-compose/k8s and leave the .../basic as it is?